pass/disk.go

```diff
@@ -125,17 +136,16 @@ func (s *diskStore) GlobSearch(query string) ([]string, error) {
 }
 
 func (s *diskStore) Open(item string) (io.ReadCloser, error) {
-	p := filepath.Join(s.path, item+".gpg")
-	if !filepath.HasPrefix(p, s.path) {
-		// Make sure the requested item is *in* the password store
-		return nil, errors.New("invalid item path")
-	}
-
-	f, err := os.Open(p)
-	if os.IsNotExist(err) {
-		return nil, ErrNotFound
+	for _, path := range s.paths {
+		path := filepath.Join(path, item+".gpg")
+		f, err := os.Open(path)
+		if os.IsNotExist(err) {
+			continue
+		}
+		// TODO this does not handle identical file names in multiple s.paths
+		return f, err
```

Comment:
And then here, when we receive an entry "github.com", the code is trying to guess, which password store it belongs to. Right now the code is simple, the first password store that contains "github.com.gpg" is the one 🙂